src/superface-test.ts

```diff
@@ -343,6 +422,25 @@ export class SuperfaceTest {
     }
   }
 
+  private composeRecordingPath(version?: string): string {
+    if (!this.recordingPath) {
```

Comment:
question: when `recordingPath` can be empty?

Reply:
It can't be empty, but as it initialises in `run()` it has to be optional and therefore I have to do this check. Not sure how can I solve it differently. 😢